plugin/metrics/prometheus/factory_test.go

```diff
@@ -34,14 +34,13 @@ func TestPrometheusFactory(t *testing.T) {
 	f := NewFactory()
 	require.NoError(t, f.Initialize(zap.NewNop()))
 	assert.NotNil(t, f.logger)
-	assert.Equal(t, "prometheus", f.options.Primary.namespace)
```

Comment:
can we keep this?

Reply:
I could compare it with the prefix as we don't have anything such as `Primary.namespace` now. Is that what you are looking for ?

Reply:
oh, I see - no, it's fine to delete.